.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,24 @@
+# .github/workflows/chromatic.yml
+
+name: 'Chromatic'
+
+on: push
+
+jobs:
+  chromatic:
+    name: Run Chromatic
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Install dependencies
+        run: npm ci
```

Comment:
I think installing the dependencies require `npm i` command.

Reply:
Rectified this